### DISCUSSION TITLE
Make/Allow Music Assistant follow user themes

### DISCUSSION THREAD

**Discussion by Mariusthvdb** [music-assistant/support#386] [Feature requests and ideas]
Currently Mass follows dark_mode or checks a theme-variable and based on that shows in either dark or regular light mode.

I am a serious user of themes, but dont like the dark-mode stylings HA has implemented so dont have these in my themes.

Of I do use color variable for background, header, color etc etc

Cut it short: please have MASS follow settings in user themes

just to have some reference for my request, this is my regular day theme, on a test page so one can see most variables live:

<img width="1479" alt="Schermafbeelding 2022-06-17 om 16 36 31" src="https://user-images.githubusercontent.com/33354141/174319827-b072c0f9-d06d-42c8-aa44-a75ceea49600.png">

apparently, Mass decides based on the header bar, this is a dark-mode theme because:

<img width="1477" alt="Schermafbeelding 2022-06-17 om 16 34 15" src="https://user-images.githubusercontent.com/33354141/174319468-db92493b-37bf-4df9-8a58-6c6603c5a6d0.png">

...

I'd be glad to send over the main color scheme, or the used theme variables used, so you could inspect and adapt..

thanks for considering

**Comment by trisweb** [comment]
+1, would be great to have user-defined themes or even just CSS customization (I haven't checked if it's possible yet; it might be).

**Comment by dreimer1986** [comment]
I would like to have such a feature, too. My reasons are maybe a bit... special though. I need to get this TamperMonkey Script (https://github.com/dreimer1986/ha_animated_backgrounds/blob/master/TamperMonkey-Scripts/Music%20Assistant%20Transparenz%20(Shadow-DOM)-1.0.txt) I made into the theme to realize that:

[Kooha-2026-06-21-14-44-54.webm](https://github.com/user-attachments/assets/5cee2778-a0da-4472-8494-3397030f4e2c)

Yes I am crazy enough to have animated backgrounds in my whole Home Assistant. And because TamperMonkey is no solution that runs everywhere, I would like to change the variables needed in a theme file.
